include/rangers-api/ApplicationCommon/FSM/GOCTinyFsm2.h

```diff
@@ -0,0 +1,96 @@
+#pragma once
+
+namespace app_cmn::fsm {
+    template<typename T>
+    class GOCTinyFsm2;
+
+    template<>
+    class GOCTinyFsm2<hh::game::GameObject> : public hh::game::GOComponent {
+    public:
+        struct Event : public hh::ut::TinyFsmEvent {
+            hh::fnd::Message* message;
+            void* updateEvent;
+        };
+        typedef hh::ut::TinyFsm<hh::game::GameObject, Event> Fsm_t;
+        typedef typename Fsm_t::State_t State_t;
+        typedef typename Fsm_t::Event_t Event_t;
+        struct alignas(8) Config {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.